api.py

```diff
@@ -179,7 +186,8 @@ def bulk():
         logging.debug("inserting %s into table %s"%(inserts,table))
 
         # We need to send the data to the correct table according to the type of data it is
-        insert_data(inserts,table)
+        print "got %s"% app.config['BULK_MYSQL_INSERT']
```

Comment:
do we need that print? I think we should use logging.info here with a more descriptive message.